modules/hibernating-cluster-hibernate.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-hibernate_{context}"]
+= Hibernating a cluster
+
+You can hibernate a cluster for up to 90 days. The cluster can recover if certificates expire while the cluster was in hibernation.
+
+.Prerequisites
+
+* The cluster has been running for at least 24 hours to allow the first certificate rotation to complete.
++
+[IMPORTANT]
+====
+If you must hibernate your cluster before the 24 hour certificate rotation, use the following procedure instead: link:https://www.redhat.com/en/blog/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms[Enabling OpenShift 4 Clusters to Stop and Resume Cluster VMs].
+====
+
+* You have taken an etcd backup.
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Confirm that your cluster has been installed for at least 24 hours.
+
+. Ensure that all nodes are in a good state by running the following command:
```

Comment:
We can merge them in one line,
```suggestion
.  Confirm that your cluster has been installed and all nodes are in a good state for at least 24 hours by running the following command:
```

Reply:
I'm not sure that it makes sense to combine these two steps, because they are two different things to check. The nodes could all be in a good state, but the cluster has only been running for a few hours.